feature_engine/base_transformers.py

```diff
@@ -144,3 +148,131 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
     # for the check_estimator tests
     def _more_tags(self):
         return _return_tags()
+
+
+class DateTimeBaseTransformer(BaseEstimator, TransformerMixin):
```

Comment:
Got it, sure

Reply:
Let's wait with this change, because I think it might be better instead of having a base class and then 3 additional classes, to have just 1 class that does everything, and then regulate their behaviour with params.